http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200609_add_runtime_disk_foreign_key_to_runtime_config.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="qi" id="add_runtime_disk_foreign_key_to_runtime_config">
+        <dropForeignKeyConstraint baseTableName="CLUSTER" constraintName="FK_CLUSTER_PERSISTENT_DISK_ID"/>
+        <dropIndex indexName="FK_CLUSTER_PERSISTENT_DISK_ID" tableName="CLUSTER"/>
+        <dropColumn tableName="CLUSTER">
```

Comment:
Are there any existing records where `CLUSTER.persistentDiskId` is populated? There may be some in dev. Wondering if we should also update those records, or if we are ok with just dropping all PD associations in the DB so far.

Reply:
Probably fine to just forget about all PD assocations